app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -556,6 +605,29 @@ export default {
   margin-bottom: 0;
 }
 
+.message-signature-wrap {
+  margin: 0 var(--space-normal);
+  padding: var(--space-small);
+  display: flex;
+  align-items: center;
```

Comment:
```suggestion
  align-items: baseline;
```

Looks a little better.
![CleanShot 2022-02-07 at 11 49 11](https://user-images.githubusercontent.com/15716057/152734881-d896734a-4669-4eeb-828c-55e109e3fba3.png)